kong/consumer_group_test.go

```diff
@@ -55,6 +55,35 @@ func TestConsumerGroupsService(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestConsumerGroupWithTags(t *testing.T) {
+	RunWhenEnterprise(t, ">=3.1.1", RequiredFeatures{})
+	require := require.New(t)
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	require.NoError(err)
+	require.NotNil(client)
+
+	cg := &ConsumerGroup{
+		Name: String("foo"),
+		Tags: StringSlice("tag1", "tag2"),
+	}
+
+	createdConsumerGroup, err := client.ConsumerGroups.Create(defaultCtx, cg)
+	require.NoError(err)
+	assert.NotNil(createdConsumerGroup)
+	require.Equal(cg.Tags, createdConsumerGroup.Tags)
+
+	createdConsumerGroup.Tags = StringSlice("tag1", "tag2", "tag3")
+	updatedConsumerGroup, err := client.ConsumerGroups.Update(defaultCtx, createdConsumerGroup)
+	require.NoError(err)
+	assert.NotNil(updatedConsumerGroup)
+	assert.Equal(updatedConsumerGroup.Tags, createdConsumerGroup.Tags)
+
+	err = client.Vaults.Delete(defaultCtx, updatedConsumerGroup.ID)
+	require.NoError(err)
```

Comment:
Do you mind moving this just below the 

```
	createdConsumerGroup, err := client.ConsumerGroups.Create(defaultCtx, cg)
	require.NoError(err)
```

and putting this into a `t.Cleanup()`? This way regardless of what happens (e.g. a `require` assertion breaks) we get that cleaned up.

Reply:
sure, fixed.